frontend/src/assets/styles/theme.js

```diff
@@ -1,6 +1,6 @@
 export const theme = {
   colors: {
     primary: '#0E74DE',
-    secondary: '#de790e',
+    secondary: 'rgb(105,106,209)',
```

Comment:
RGB color it's #696AD1 in HEX, I think one convention of color can be better.

Reply:
As I have written in the previous comment, the colours need to be unified and we should discuss which of them should be used. That's why most of them aren't in the theme.